app/src/debug/java/com/artemzin/qualitymatters/ui/presenters/DeveloperSettingsPresenter.java

```diff
@@ -41,7 +41,7 @@ public void syncDeveloperSettings() {
     }
 
     public void changeStethoState(boolean enabled) {
-        if (developerSettingsModel.isStethoEnabled() == enabled) {
+        if (!developerSettingsModel.hasStethoStateChanged(enabled)) {
```

Comment:
it sort of removes logic, but logic is really is just boolean comparison that makes model code weird…